benchmarks/build.gradle.kts

```diff
@@ -13,7 +13,11 @@ plugins {
 }
 
 kotlin {
-    jvm()
+    jvm {
```

Comment:
starting from gradle 8 you also need to add foojay toolchain, so JDK will be automatically downloaded if not present
more info: https://github.com/gradle/foojay-toolchains

Reply:
I'm not 100% sure if it should be enabled, filed an issue to discuss it and enable later: https://github.com/Kotlin/kotlinx-io/issues/184

Reply:
TBH, I don't really understand, why it should not be enabled - it simplifies contributors setup, as not all of them could have JDK 8 installed, but up to you, for me it's not an issue